### PR TITLE
doc: Add the integration guide for InfluxDB Cloud 2.0

### PR DESCRIPTION
**Pull request by nejraselimovic**
#### Summary
Adding the guide for integrating TTS with InfluxDB Cloud 2.0 with MQTT server and Telegraf agent.

#### Testing
Tested with Telegraf agent and TTS running locally.

#### Checklist
- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [x] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [ ] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: doc/content/integrations/influxdb-cloud/influxdb-cloud-setup/_index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Go to the **Telegraf** tab and select **Create Configuration**. 
+
+When asked **What do you want to monitor?**, select **System**. 
+
+{{< figure src="monitoring-system.png" alt="Selecting to monitor a system" >}}
+
+Name your configuration, select **Create and Verify** and then **Finish**.
+
+Once the configuration is created, you can simply click on its name in **Telegraf** tab and download it.
```
Comment (laurensslats) [inline doc/content/integrations/influxdb-cloud/influxdb-cloud-setup/_index.md]:
```suggestion
Once the configuration is created, you can simply click on its name in the **Telegraf** tab and download the configuration file.
```

File: doc/content/integrations/influxdb-cloud/_index.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: "InfluxDB Cloud 2.0"
+description: ""
+weight: 
+---
+
+[InfluxDB Cloud 2.0](https://v2.docs.influxdata.com/v2.0/get-started/) is a serverless real-time monitoring platform, based on data store specifically created for working with time series data. It combines data storage, user interface, visualization, processing, monitoring and alerting into one cohesive system.
```
Comment (benolayinka) [inline doc/content/integrations/influxdb-cloud/_index.md]:
```suggestion
[InfluxDB Cloud 2.0](https://v2.docs.influxdata.com/v2.0/get-started/) is a serverless real-time monitoring platform specifically created for working with time series data. It combines data storage, user interface, visualization, processing, monitoring and alerting into one cohesive system. 
```

File: doc/content/integrations/influxdb-cloud/telegraf-setup/_index.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+{{< figure src="tts-mqtt-info.png" alt="The Things Stack MQTT server info" >}}
+
+Update the previously downloaded Telegraf configuration with the following lines and modify them according to your MQTT server info:
+
+```bash
+[[inputs.mqtt_consumer]]
+#
+# MQTT broker URLs to be used. The format is scheme://host:port, schema can be tcp, ssl, or ws.
+  servers = ["tcp://127.0.0.1:1883"]
```
Comment (benolayinka) [inline doc/content/integrations/influxdb-cloud/telegraf-setup/_index.md:21]: Shouldn't the schema be `mqtt:` or `mqtts:` ? @adriansmares

**Comment by benolayinka** [comment]
pushed some minor grammar edits @nejraselimovic , looks good to me otherwise but let's wait for @adriansmares to confirm the mqtt schema

File: doc/content/integrations/influxdb-cloud/telegraf-setup/_index.md (hunk 3 above)
Comment (adriansmares) [reply doc/content/integrations/influxdb-cloud/telegraf-setup/_index.md:21]: Our pub/sub form validation requires `mqtt` or `mqtts`, but the one Telegraf uses accepts `tcp`. You can see this in the comment above the line.

Comment (benolayinka) [reply doc/content/integrations/influxdb-cloud/telegraf-setup/_index.md:21]: ah but they all resolve to the same type of tls request?

Comment (adriansmares) [reply doc/content/integrations/influxdb-cloud/telegraf-setup/_index.md:21]: Yes, fundamentally `tls`, `ssl`, `tcps` and `mqtts` are equivalent for our pub/sub driver, while the one Telegraf uses accepts only `tls` and `ssl` (but they are still equivalent).

SSL is the precursor to TLS, and while technologically speaking they are different, the terms are used interchangeability nowadays and they both refer to TLS. I personally have yet to meet a program/library that interprets `tls://` and `ssl://` differently.

You may say that there are a lot of terms that actually mean the same thing and this leads to confusion, I couldn't agree more, but this is the way TLS and SSL are now surviving in our lingo.

**Pull request merged**